core/server/proxy/src/main/java/alluxio/web/ProxyWebServer.java

```diff
@@ -102,6 +106,12 @@ public ProxyWebServer(String serviceName, InetSocketAddress address,
         .register(S3RestExceptionMapper.class);
 
     mFileSystem = FileSystem.Factory.create(Configuration.global());
+    long rate =
+        Configuration.getLong(PropertyKey.PROXY_S3_GLOBAL_READ_RATE_LIMIT_MB) * Constants.MB;
```

Comment:
I think we need to be careful there, it might be larger than long.MAX_VALUE;

Reply:
The check level of properties is set to `ConsistencyCheckLevel.ENFORCE`, if users set error format, the proxy will start failed, and throw an exception.

Reply:
I mean here is `rate = long * 1024 * 1024`, it may over long.MAX_VALUE(2,147,483,647);

Reply:
I'm sorry, I fixed it now.